### PR TITLE
February 6, 2024 release notes

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_help/release_notes.md [hunk 1]
```diff
@@ -54,6 +54,111 @@ guide_featured_list:
 > <br>
 > For more information on any of the updates listed in this section, reach out to your account manager or [open a support ticket]({{site.baseurl}}/help/support/). You can also check out [our SDK Changelogs]({{site.baseurl}}/developer_guide/platform_integration_guides/sdk_changelogs/) to see more information on our monthly SDK releases, updates, and improvements.
 
+## February 6, 2024 release
+
+### Braze privacy manifest
+
+Braze has released our own privacy manifest, along with new flexible APIs that automatically reroute declared tracking data to dedicated `-tracking` endpoints. For more information, see [Braze's privacy manifest]({{site.baseurl}}/docs/developer_guide/platform_integration_guides/swift/privacy_manifest).
```
Comment (lydia-xie) [inline _docs/_help/release_notes.md:61]:
```suggestion
Braze has released our own privacy manifest, along with new flexible APIs that automatically reroute declared tracking data to dedicated `-tracking` endpoints. For more information, see [Braze's privacy manifest]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/privacy_manifest).
```

File: _docs/_help/release_notes/2024/2_6_24.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+noindex: true
+page_type: update
+description: "This article contains release notes for 2/6/2024."
+---
+
+# February 6, 2024 release
+
+## Braze privacy manifest
+
+Braze has released our own privacy manifest, along with new flexible APIs that automatically reroute declared tracking data to dedicated `-tracking` endpoints. For more information, see [Braze's privacy manifest]({{site.baseurl}}/docs/developer_guide/platform_integration_guides/swift/privacy_manifest).
```
Comment (lydia-xie) [inline _docs/_help/release_notes/2024/2_6_24.md:13]:
```suggestion
Braze has released our own privacy manifest, along with new flexible APIs that automatically reroute declared tracking data to dedicated `-tracking` endpoints. For more information, see [Braze's privacy manifest]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/privacy_manifest).
```

File: _docs/_help/release_notes.md [hunk 3]
```diff
@@ -54,6 +54,111 @@ guide_featured_list:
 > <br>
 > For more information on any of the updates listed in this section, reach out to your account manager or [open a support ticket]({{site.baseurl}}/help/support/). You can also check out [our SDK Changelogs]({{site.baseurl}}/developer_guide/platform_integration_guides/sdk_changelogs/) to see more information on our monthly SDK releases, updates, and improvements.
 
+## February 6, 2024 release
+
+### Braze privacy manifest
+
+Braze has released our own privacy manifest, along with new flexible APIs that automatically reroute declared tracking data to dedicated `-tracking` endpoints. For more information, see [Braze's privacy manifest]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/privacy_manifest).
```
Comment (bre-fitzgerald) [inline _docs/_help/release_notes.md:61]: Braze as a possessive! 🙅 

```suggestion
Braze has released our own privacy manifest, along with new flexible APIs that automatically reroute declared tracking data to dedicated `-tracking` endpoints. For more information, see the [Braze privacy manifest]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/privacy_manifest).
```

File: _docs/_help/release_notes.md [hunk 4]
```diff
@@ -54,6 +54,111 @@ guide_featured_list:
...
+- [Return a list of past sync statuses]({{site.baseurl}}/api/endpoints/cdi/get_job_sync_status/) for a given integration.
+- [Trigger a sync]({{site.baseurl}}/api/endpoints/cdi/post_job_sync/) for a given integration.
+
+#### Braze Cloud Data Ingestion (CDI) support for Databricks
+
+Braze CDI support for catalogs is now available for [Databricks sources]({{site.baseurl}}/user_guide/data_and_analytics/cloud_ingestion/sync_catalogs_data/#step-2-integrate-cloud-data-ingestion-with-catalog-data).
+
+#### Manual Swift SDK integration
+
+We added the [Manual integration]({{site.baseurl}}}/developer_guide/platform_integration_guides/swift/initial_sdk_setup/installation_methods/manual_integration/article) article to the integration guides to describe how to manually integrate the Swift SDK without the use of a package manager.
```
Comment (josh-mccrowell-braze) [inline _docs/_help/release_notes.md:90]: This link seems broken.

Comment (rachel-feinberg) [reply _docs/_help/release_notes.md:90]: Agh, looks like the "article" at the end should be removed. I did that but the link still doesn't work. 🤔 I'll push the commit and see what happens

Comment (rachel-feinberg) [inline _docs/_help/release_notes.md:90]:
```suggestion
We added the [Manual integration]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/initial_sdk_setup/installation_methods/manual_integration/article) article to the integration guides to describe how to manually integrate the Swift SDK without the use of a package manager.
```

File: _docs/_help/release_notes.md [hunk 5]
```diff
@@ -54,6 +54,111 @@ guide_featured_list:
...
+- [Return a list of past sync statuses]({{site.baseurl}}/api/endpoints/cdi/get_job_sync_status/) for a given integration.
+- [Trigger a sync]({{site.baseurl}}/api/endpoints/cdi/post_job_sync/) for a given integration.
+
+#### Braze Cloud Data Ingestion (CDI) support for Databricks
+
+Braze CDI support for catalogs is now available for [Databricks sources]({{site.baseurl}}/user_guide/data_and_analytics/cloud_ingestion/sync_catalogs_data/#step-2-integrate-cloud-data-ingestion-with-catalog-data).
+
+#### Manual Swift SDK integration
+
+We added the [Manual integration]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/initial_sdk_setup/installation_methods/manual_integration/article) article to the integration guides to describe how to manually integrate the Swift SDK without the use of a package manager.
```
Comment (rachel-feinberg) [inline _docs/_help/release_notes.md:90]:
```suggestion
We added the [Manual integration]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/initial_sdk_setup/installation_methods/manual_integration) article to the integration guides to describe how to manually integrate the Swift SDK without the use of a package manager.
```

File: _docs/_help/release_notes/2024/2_6_24.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+- [Return a list of past sync statuses]({{site.baseurl}}/api/endpoints/cdi/get_job_sync_status/) for a given integration.
+- [Trigger a sync]({{site.baseurl}}/api/endpoints/cdi/post_job_sync/) for a given integration.
+
+### Braze Cloud Data Ingestion (CDI) support for Databricks
+
+Braze CDI support for catalogs is now available for [Databricks sources]({{site.baseurl}}/user_guide/data_and_analytics/cloud_ingestion/sync_catalogs_data/#step-2-integrate-cloud-data-ingestion-with-catalog-data).
+
+### Manual Swift SDK integration
+
+We added the [Manual integration]({{site.baseurl}}}/developer_guide/platform_integration_guides/swift/initial_sdk_setup/installation_methods/manual_integration) article to the integration guides to describe how to manually integrate the Swift SDK without the use of a package manager.
```
Comment (rachel-feinberg) [inline _docs/_help/release_notes/2024/2_6_24.md:42]:
```suggestion
We added the [Manual integration]({{site.baseurl}}/developer_guide/platform_integration_guides/swift/initial_sdk_setup/installation_methods/manual_integration) article to the integration guides to describe how to manually integrate the Swift SDK without the use of a package manager.
```

**Review by bre-fitzgerald — APPROVED** [review]
lgtm!

**Pull request merged**